DiagManager/CustomDiagnostics/SQL Base/Multicopy.ps1

```diff
@@ -0,0 +1,22 @@
+﻿param(
+  [Parameter(Position=0)]
+  [string]$sourcePath,
+  [Parameter(Position=1)]
+  [string]$destinationPath,
+  [Parameter(Position=2)]
+  [string]$serverName
+)
+
+
+Get-ChildItem $sourcePath -FILE | ForEach-Object { 
+		$newfileName = $serverName + "_" + $_.Name
+ 
+		$newfileName = Join-path $destinationPath $newfileName
+
+	Try { 
+				Copy-Item -Path $_.FullName -Destination $newFileName -ErrorAction Stop
+		}		
+	Catch {
+				$_.Exception.Message | Out-File -FilePath "c:\temp\error.txt" -Append
```

Comment:
I would keep the $_.Exception.Message part and either send it to the console with **Write-Output** , or if you want to send to a file, replace the c:\temp with a file in the internal folder %internal_output_name% -is variable for it so better pass it as a parameter, The file can be named something like  $serverName + "_" + TaskName (best to pass this as a parameter into the script)